### PR TITLE
changelog 14

### PR DESCRIPTION
**Pull request by pomdtr**

### CODE REVIEW

File: src/pages/changelog/post-14.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Revamped Build Pipeline"
+date: "12 Apr 2023"
```
Comment (BetaHuhn) [inline src/pages/changelog/post-14.mdx:3]:
```suggestion
date: "27 Apr 2023"
```

File: src/pages/changelog/post-14.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+layout: "@changelogs"
+description: "We revamped our build pipeline to make it more reliable."
+---
+
+## New Build Pipeline
+
+### Improved Logs
+
+We reworked the build logs to be easier to read and more informative.
```
Comment (BetaHuhn) [inline src/pages/changelog/post-14.mdx:12]: Would be good if we can add a screenshot here

File: src/pages/changelog/post-14.mdx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Revamped Build Pipeline"
+date: "12 Apr 2023"
+layout: "@changelogs"
+description: "We revamped our build pipeline to make it more reliable."
+---
+
+## New Build Pipeline
```
Comment (BetaHuhn) [inline src/pages/changelog/post-14.mdx:8]:
```suggestion
## New Build Pipeline

We have been working hard on a revamped build pipeline that is more reliable, easier to maintain and provides a few improvements for developers building Space apps as well.
```

**Review by BetaHuhn — COMMENTED** [review]
Just a few suggestions

File: src/pages/changelog/post-14.mdx (hunk 2 above)
Comment (pomdtr) [reply src/pages/changelog/post-14.mdx:14]: Not sure it is really possible to demonstrate it without posting a full-page image.

I'm not sure as a user I would be able to notice the improvements just by looking at this:

image

Comment (BetaHuhn) [reply src/pages/changelog/post-14.mdx:14]: Doesn't have to be a real screenshot I guess, can also just be a code block that shows the main changes:

```
📦  Pushing your code (7 files) & running build process...

2023/04/27 15:37:05 Parsing Spacefile...
2023/04/27 15:37:05 Building micro client...
2023/04/27 15:37:05 [client] Installing dependencies with npm
2023/04/27 15:37:09 [client] Zipping dependencies
2023/04/27 15:37:09 [client] Zipping code
2023/04/27 15:37:09 
2023/04/27 15:37:09 Building micro server...
2023/04/27 15:37:09 [server] Installing dependencies with npm
2023/04/27 15:37:11 [server] Running command: npm run build
2023/04/27 15:37:11 [server] ...
2023/04/27 15:37:11 [server] Zipping code
2023/04/27 15:37:11 
2023/04/27 15:37:11 All micros built successfully
2023/04/27 15:37:11 Total Size 4.62 MB
2023/04/27 15:37:18 Created revision: dory-weyi

💻  Updating your Builder instance with the new revision...

2023/04/27 15:37:23 starting update...
2023/04/27 15:37:23 fetching release configuration...
2023/04/27 15:37:23 allocating resources..
2023/04/27 15:37:23 configuring resource 1 out of 2...
2023/04/27 15:37:43 configuring resource 2 out of 2...
2023/04/27 15:38:02 completed...
                                                                    
🎉  Successfully pushed your code and updated your Builder instance!
Builder instance: https://example-v7468792.deta.app
```

**Review by backslash-zero — APPROVED** [review]
All good for me! Will add it to next week's newsletter! Good work guys!

**Review by BetaHuhn — APPROVED** [review]

**Pull request merged**